lambda_src/response_storage.py

```diff
@@ -0,0 +1,47 @@
+import os
+from typing import Dict, Text
+
+import boto3
+
+AWS_REGION = os.environ[
+    'AWS_REGION'
+]
+DYNAMODB_TABLE = 'geff-requests'
```

Comment:
users might want to customize the table they use for locks. by default let's not call it "GEFF requests", as it's up to the driver code whether there's a single request, or multiple requests, or something else entirely here (e.g. sending an email is usually called a request).

Reply:
Would it be okay to have it passed as an environment variable?

Reply:
not 100% sure but I think that makes sense -- and the user sets it in Terraform?